IFC4x3/Properties/w/WaterLevelType_1c5OF998b1_8SQzyRB0_vU/Documentation.md

```diff
@@ -0,0 +1 @@
+Type of measured water level (free groundwater / rise to this measured level).
```

Comment:
Having trouble understanding this description. What would be an example of a `WaterLevelType`?

Cannot make suggestion if I do not understand.

Reply:
> Having trouble understanding this description. What would be an example of a `WaterLevelType`?
> 
> Cannot make suggestion if I do not understand.

Ping @j-weil.

Reply:
Water level types would be
"free" or "piezometric / confined" when the groundwater body is limited on top by a seal and the water will rise in a pipe that sticks into it. Helpful?

Reply:
proposal: IsPiezometric : IfcBoolean - Defines whether hight measurements are made vs the piezometric surface (true) or the surface of the groundwater (false)

@j-weil: Please confirm!

Reply:
Fixed in [1b1bc72](https://github.com/bSI-InfraRoom/IFC-Specification/pull/585/commits/1b1bc72aa23d64541fd411ac8250a25a69c46a2d)